packages/cloud-cognitive/src/components/BreadcrumbWithOverflow/BreadcrumbWithOverflow.js

```diff
@@ -51,6 +51,20 @@ export let BreadcrumbWithOverflow = ({
   const internalId = useRef(uuidv4());
   const [childArray, setChildArray] = useState([]);
 
+  const getItemProps = (item) => {
```

Comment:
I'm not sure this is the best approach. It's quite a lot of work for very little use. Have you considered `innerText` for extracting a title? Perhaps we could discuss.

Reply:
How about using ReactDomServer.renderToString